cypress/support/testHelper.js

```diff
@@ -37,15 +37,15 @@ export const checkLinkStyling = position => {
   shouldContainStyles(link, 'color', 'rgb(255, 255, 255)');
   link.focus();
   shouldContainStyles(
-    link,
-    'text-decoration',
-    'underline solid rgb(255, 255, 255)',
+    link.get('span').eq(position),
```

Comment:
I'd suggest creating a constant for this element before this `shouldContainStyles` and to use this constant here and for the assertion below.